.github/workflows/dists.yml

```diff
@@ -55,10 +55,10 @@ jobs:
 
       - name: Build wheels
         env:
-          # Skip building on Python 2.7 and PyPy
+          # Skip building on Python 2.7
           # Additionally, skip 32-bit Windows for now as MSVC needs seperate setup with different toolchain to do this
           # Refer: https://cibuildwheel.readthedocs.io/en/stable/cpp_standards/#windows-and-python-27
-          CIBW_SKIP: cp27-* pp* *-win32 cp35-*
+          CIBW_SKIP: cp27-* *-win32 cp35-*
```

Comment:
You can remove the skipping of cp27 & cp35 since cibuildwheel 2.0 (in fact, it should be printing a warning stating these are not needed).

Reply:
yeah that's for the suggestion, but I think we intentionally skipped cp27, because the underlying C++ library won't compile and python2.7 is reaching end of life. But I cannot remember clearly why we skipped cp35 though. But I am pretty sure certain it was because an issue, though I am not sure what the issue was. Do you have a need for these two to work? If so I can take a look later probably.

Reply:
Cibuildwheel 2.0 dropped Python 2.7 and Python 3.5, so there's no need to skip them anymore, is what I meant. :)

Reply:
Ah make sense! Thanks for letting me knnow.